entry_types/scrolled/package/src/editor/views/EditSectionTransitionEffectView.module.css

```diff
@@ -16,68 +26,52 @@
 }
 
 .active {
-    background: #6c757d !important;
+    background: selectionColor !important;
 }
 
-.active label {
+.active div {
     color: #ffffff;
 }
 
 .transition {
     padding: 15px 0;
     height: 60px;
-    display: flex;
+    display: flex !important;
     align-items: center;
-    border-top: 1px solid #808080;
+    border-top: 1px solid #ffffff;
     background: #e5e5e5;
 }
-.transition:hover,
-.transition input[type="radio"]:checked + label {
+.transition:hover {
      background: #ddd;
 }
 
 
-/*Fade*/
-.transition:hover .fade .upper_section {
-    visibility: hidden;
-    opacity: 0.5;
-}
-
-/*Fade Background Upper*/
+/*Fade AND Fade Background Upper*/
+.transition:hover .fade .upper_section,
 .transition:hover .fadeBg .upper_section {
-    visibility: hidden;
-    transform: translateY(-100%);
+    animation: Fade 2s linear infinite;
 }
 
-/*face Background Lower*/
-.transition:hover .fadeBg .lower_section {
-    transform: translateY(-100%);
-}
-
-/*Scroll and Scroll Over*/
+/*Scroll, Scroll Over, Reveal, AND Fade Background Lower*/
 .transition:hover .scroll .upper_section,
 .transition:hover .scroll .lower_section,
-.transition:hover .scrollOver .lower_section {
-    animation: Scroll 2s ease infinite;
-}
-
-/*Reveal*/
-.transition:hover .reveal .upper_section {
-    animation: Reveal 2s ease infinite;
+.transition:hover .scrollOver .lower_section,
+.transition:hover .reveal .upper_section,
+.transition:hover .fadeBg .lower_section {
+    animation: Translate 2s linear infinite;
```

Comment:
Changed the Scroll keyframe name to more generic as Translate since a lot of different transitions use the same.
I did same for the Fade.